### DISCUSSION TITLE
With which corpora is the French accurate pipeline (fr_dep_new_trf) trained ?

### DISCUSSION THREAD

**Discussion by XavBeckers** [explosion/spaCy#8032] [Language Support]
Hello everyone ! 
In the context of my graduate work, which is about a linguistic evaluation of syntactic parsing results in french, I'm searching for information about spacy's french pipeline. Indeed, we're considering the interest of a comparison of the efficiency of pipelines trained on two label sets (UD and SUD), and perform a qualitative analysis of the results, more on the linguistic side than the statistical side.
For this purpose, i'd need to know if the corpora used to train the original pipeline is known and available, in order to train a custom pipeline using SUD tag set onto the same data (annotated with SUD) and ensure the comparison.
Do someone have any information about training data of this particular pipeline ? 
Have a good afternoon !

**Comment by adrianeboyd** [comment]
Hi, the sources are included in `meta.json` in the model package and shown for each model on the models page here: https://spacy.io/models/fr

In the v3.0.0 releases, the vectors information was accidentally left out of the metadata. The vectors for French are fastText cbow vectors trained on OSCAR (Common Crawl) and Wikipedia, published by us (Explosion).
